lib/main.dart

```diff
@@ -2,14 +2,15 @@ import 'package:firebase_core/firebase_core.dart';
 import 'package:flutter/material.dart';
 import 'package:intl/date_symbol_data_local.dart';
 import 'package:zachranobed/firebase/firebase_options.dart';
+import 'package:zachranobed/firebase/notifications.dart';
 import 'package:zachranobed/services/ioc_container.dart';
 import 'package:zachranobed/ui/widgets/app_root.dart';
 
 void main() async {
   WidgetsFlutterBinding.ensureInitialized();
   await Firebase.initializeApp(options: DefaultFirebaseOptions.currentPlatform);
-
   IoCContainer.setup();
+  await Notifications().initNotifications();

```

Comment:
Nevim jestli chces se ptat na prava ihned po zapnutí. Na iOS je vyžadována většinou samostatná screen kde je potřeba vysvětlit why. Často to projde i bez toho, ale good practice je request permissions udělat s nějakým krokem